### DISCUSSION TITLE
Getting pubs to index in Google Scholar: sharing tips, experience, knowledge

### DISCUSSION THREAD

**Discussion by daaronr** [knowledgefutures/pubpub#2863] [General]
As part of the [The Unjournal's](https://unjournal.org) success model/ToC, it's very important to get (all of) [our outputs](https://unjournal.pubpub.org/) indexed in Google Scholar.  The sooner this can be done in the publication cycle the better (ideally, during the period that traditional journal referees are also evaluating these papers.) 

Context: We do journal-independent evaluation, so outputs are evaluations ('reviews') we have commissioned, as well as authors' responses and manager summaries. 

I believe we are following the recommended approach: 
- getting DOI's for each output (or for the 'summary' document, with the others connected, which should accomplish this)
- connecting to and referencing the original paper being evaluated
- attaching ORCID ID's to all authors/evaluators involved where possible 
- labeling and tagging these as much as possible 
- PubPub has asked Google to index our work 

I think we've been doing this for about 6 months. However, so far [only one item](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=source%3A%22unjournal%22&btnG=), an authors' response to an evaluation of a traditionally published work, is indexed. 

*How have others fared with this? Any suggestions for 'what works'?*  Thanks!

**Comment by tefkah** [comment]
Hi! I run the [Journal of Trial and Error](https://journal.trialanderror.org/) (and also work with PubPub), and [our articles do consistently get indexed by Scholar](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=source%3A%22journal+of+trial+%26+error%22&btnG=). 
[This being the most recent one](https://journal.trialanderror.org/pub/manifesto-rewarding-recognizing/release/1), from Mid-August.  It usually takes a month or so for articles to show up in Scholar.

Other than what you have already mentioned and what [Google suggests](https://scholar.google.com/intl/en/scholar/inclusion.html#crawl) (although I feel this is not kept up to date that much), I can recommend two things:
1. Have an abstract. If you title the first heading 1 of your Pub "Abstract", it will become a kind of special container which looks nice, but also 
	- Puts your abstract in the metadata of the page through a `<meta name="citation_abstract">` tag.
	- fulfills the requirement [Google lists here](https://scholar.google.com/intl/en/scholar/inclusion.html#content) of your abstract being the first thing shown to a user.
2. Include a heading with the name "References" or "Bibliography". [Google mentions this here as the 3rd thing to do](https://scholar.google.com/intl/en/scholar/inclusion.html#indexing). This again signals "this is academic content please look at me"

 
Not sure how easy it is to do these, as you are posting evaluations, but at least the references being a separate header might help.
We also run a [blog which does not have peer reviewed content and no abstracts](https://blog.trialanderror.org), but the articles that do have references labeled like this [do get indexed](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=source%3A%22blog+of+trial+%26+error%22&btnG=) (even some without references for some reason).

That said, I don't think there's a silver bullet for this, it's a lot of guesswork! 

The only other thing I can think of is to make sure your pages are being indexed by Google. If you get a custom domain with pubpub (like we have) you can register it to the Google Search console and check this, and, if needed, manually submit articles to be indexed. This doesn't guarantee they show up in Scholar, but I don't think articles can show up in Scholar if they aren't indexed by Google in the first place.

That's most of what I've come to learn, hope it's useful!
